npm/vite-dev-server/cypress/support/e2e.ts

```diff
@@ -17,4 +17,4 @@
 import '@packages/frontend-shared/cypress/e2e/support/e2eSupport'
 
 // Alternatively you can use CommonJS syntax:
-// require('./commands')
+require('./commands')
```

Comment:
I don't like mixing these. Why would you require instead of import? Not gonna block but stick to one. require is nonstandard in the browser.

Reply:
I can update this. I had just uncommented what was commented.

Reply:
Updated here: [`dd566a0` (#21599)](https://github.com/cypress-io/cypress/pull/21599/commits/dd566a0562093ca231c4270c3cc76d873e013833)